Src/UIComponents/TileAPIKey/TileAPIKey.css

```diff
@@ -0,0 +1,44 @@
+.api-key-tile {
+    background: inherit;
+    display: flex;
+    flex-direction: column;
+    font-weight: 500;
+    width: 37rem;
+    letter-spacing: 0.035rem;
+    margin-left: 1.8rem;
+}
+.api-key-tile:last-child {
+    margin-bottom: 1rem;
+}
+.api-key-tile__header {
+    font-size: 0.8rem;
+    color: rgba(74, 72, 78, 0.7);
+    background: inherit;
+    margin: 1.5rem 0 1rem 0;
```

Comment:
shouldn't the header get bigger font than the rest of the texts?

Reply:
That header declaration concerns the small text 'API KEY' in the top left corner of the component. 
I did it in line with design. Maybe it would look better but I'm not sure if the rest of components follows that design.